cmd/proxy-server/main.go

```diff
@@ -389,13 +395,22 @@ func run(log *logrus.Entry) error {
 	}
 	defer storageConn.Close()
 
+	proxyConn, err := grpc.Dial(proxyAddr,
```

Comment:
What is this being used for? It looks like a grpc connection to itself. The client for this connection isn't used in the `refreshAdminTokenHandler`.

Reply:
Removed this.